services/terraformer/templates/oci.tpl

```diff
@@ -163,10 +163,18 @@ resource "oci_core_instance" "{{ $node.Name }}" {
         - netfilter-persistent save
         {{- if not $nodepool.IsControl }}
         # Mount volume
-        - mkdir -p /opt/claudie/data
-        - mkfs.xfs /dev/sdb
-        - mount /dev/sdb /opt/claudie/data
-        - echo "/dev/sdb /opt/claudie/data xfs defaults 0 0" >> /etc/fstab
+        - |
+          sleep 50
```

Comment:
could this be replaced by a `while` loop waiting e.g. `1s` before each check?
also, which is better: to stand up the machine without the requested disk or have it stuck in this stage?